docs/README.md

```diff
@@ -1,19 +1,23 @@
 # About
 
-This is the main documentation site for DPL repositories (Danish
-Public Libraries). It contains general DPL and cross repository
-documentation and consolidates documentation from individual
-repositories.
+This is the main documentation site for [Danish Public
+Libraries](https://detdigitalefolkebibliotek.dk/) projects. These
```

Comment:
Please consider readding `(DPL)` somewhere to make the relationship between the organisation and the abbreviation clear.

Reply:
Will do.